package.json

```diff
@@ -5,7 +5,8 @@
   "version": "0.10.2",
   "dependencies": {
     "sockjs": "0.3.15",
-    "sugar": "1.4.1"
+    "sugar-deprecated": "0.0.1",
+    "object.values": "1.0.3"
```

Comment:
We're not really decreasing our dependency count with this... can we just manually shim it in `tools.js`?